diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -965,7 +965,15 @@ int DifferentialFunctionComparator::cmpGlobalValues(GlobalValue *L,
                 }
             }
             return 0;
-        } else
+            // Externally defined constants (those without initializer
+            // and with different names) need to have their definitions linked.
```

Comment:
The comment should go below the condition (so that it is inside the block that it belongs to).